src/main/scala/sbtdocker/DockerPush.scala

```diff
@@ -44,5 +47,20 @@ object DockerPush {
     val process = Process(command)
     val exitValue = process ! processLog
     if (exitValue != 0) sys.error("Failed to push")
+
+    val PushedImageDigestSha256 = ".* digest: sha256:([0-9a-f]+) .*".r
+
+    val imageId = lines.collect {
+      case PushedImageDigestSha256(digest) => ImageDigest("sha256", digest)
+    }.lastOption
+
+    imageId match {
+      case Some(id) =>
+        imageName -> id
+      case None =>
+        throw new DockerPushException("Could not parse Docker image id")
```

Comment:
This block refers to "image id" at some places can we update this to refer to "image digest" instead?

After this it can be merged

Reply:
fixed!